srcpkgs/dnsdist/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'dnsdist'
+pkgname=dnsdist
+version=1.6.0
+revision=1
+build_style=gnu-configure
+configure_args="--with-pic --with-gnu-ld --with-libsodium --with-re2
+ --with-net-snmp --with-libcap --with-libssl --enable-dnscrypt
+ --with-libcrypto=${XBPS_CROSS_BASE}/usr --with-service-user=_dnsdist
+ --with-service-group=_dnsdist --sbindir=/usr/bin --with-gnutls --with-lmdb
+ --enable-dns-over-tls --enable-dns-over-https --sysconfdir=/etc/dnsdist"
```

Comment:
Does it make sense to build with openssl *and* gnutls?

Reply:
I never thought so, and personally I dislike GNUtls, but that's how Debian & Ubuntu built dnsdist when I first opened this PR.  I just double checked and sure enough, Debian agrees with you.  They dropped GNUtls support, so I just did the same.